conans/client/remote_manager.py

```diff
@@ -97,6 +97,9 @@ def get_recipe(self, conan_reference, dest_folder, remote):
         files = unzip_and_get_files(zipped_files, dest_folder, EXPORT_TGZ_NAME)
         # Make sure that the source dir is deleted
         rmdir(self._client_cache.source(conan_reference), True)
+        for dirname, _, files in os.walk(dest_folder):
```

Comment:
Dangerous, overwriting files which is the variable used to be returned from previous call. I would rename it, also in ``get_package()``. Can you make the change, or do you want me to send a PR for your PR? :)